packages/core/src/main/conversation/message/MessageService.ts

```diff
@@ -80,10 +80,7 @@ export class MessageService {
       cipherText = externalPayload.cipherText;
     }
 
-    const {encrypted, missing} = await this.cryptographyService.encrypt(plainTextPayload, recipients);
-    const encryptedPayload = Object.keys(missing).length
-      ? await this.reencryptAfterMismatch({missing, deleted: {}}, encrypted, plainText)
-      : encrypted;
+    const encryptedPayload = await this.cryptographyService.encrypt(plainTextPayload, recipients);
```

Comment:
Now the `encrypt` method automatically encrypts for clients that do not have a session yet (no more missing recipients)